### PR TITLE
cleanup and upgrade

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
https://github.com/openmicroscopy/ome-documentation/pull/1642 follow up

see:
- https://github.com/openmicroscopy/openmicroscopy/pull/5153
- https://github.com/openmicroscopy/management_tools/pull/328

check documentation

### CODE REVIEW

File: ansible/templates/common/virtualenv.sh.j2 [hunk 1]
```diff
@@ -1,45 +1,12 @@
...
-fi
-{% endif %}
-
-{% if ice_version == '3.5' %}
-pip install -r {{ omero_user_home_dir }}/OMERO.py/share/web/requirements-py27-nginx.txt
-pip install -r {{ omero_user_home_dir }}/OMERO.py/share/web/requirements-common-trial.txt
-{% else %}
-pip install -r {{ omero_user_home_dir }}/OMERO.py/share/web/requirements-py27-trial.txt
-{% endif %}
+{{ virtualenv_path }}/bin/pip install --upgrade omego
```
Comment (jburel) [inline ansible/templates/common/virtualenv.sh.j2]: add extra line

File: ansible/templates/centos7/deps.sh.j2 [hunk 2]
```diff
@@ -10,14 +10,21 @@ yum -y install epel-release
 
 yum -y install git
 
-yum -y install gcc gcc-c++ zlib-devel
+yum -y install python-pip
 
+{% if system_site_packages %}
+yum -y install \
+    python-pillow \
+    numpy
```
Comment (jburel) [inline ansible/templates/centos7/deps.sh.j2]: Do we want to add ``python-requests``?
similar to https://github.com/openmicroscopy/ansible-role-omero-web-runtime/pull/3/files

File: ansible/templates/common/websession_configure.sh.j2 [hunk 3]
```diff
@@ -8,14 +8,14 @@ set -e -u -x
...
+source {{ virtualenv_path }}/bin/activate
 {% else %}
 set +u
-source {{ omero_user_home_dir }}/omerowebvenv/bin/activate
+source {{ virtualenv_path }}/bin/activate
 set -u
 {% endif %}
 
-pip install django-redis>=4.4
+pip install --upgrade django-redis>=4.4
```
Comment (jburel) [inline ansible/templates/common/websession_configure.sh.j2]: why not using ``requirements-redis.txt``?

**Comment by jburel** [comment]
Upgrading to Ubuntu 16.04 means that the following lines
``
apt-get -y install software-properties-common
add-apt-repository -y ppa:nginx/stable
``
can be removed from ``nginx.sh.j2``
1.10 will be installed

File: ansible/templates/centos7/deps.sh.j2 (hunk 2 above)
Comment (aleksandra-tarkowska) [inline ansible/templates/centos7/deps.sh.j2]: not really, that is not a web requirement

**Comment by aleksandra-tarkowska** [comment]
thx for the comments I will update once we sort out https://trello.com/c/IpCSdO5Q/731-default-ice-version

**Comment by jburel** [comment]
thinking back, you should also remove Ubuntu 14.04 that I introduced in the readme and only keep Ubuntu 16.04 which the recommended version

**Comment by aleksandra-tarkowska** [comment]
@jburel removing the above from ubuntu cased ```/tmp/omeroweb-install-test/ubuntu-ice3.6/run: line 7: cron: command not found```. should I revert any if https://github.com/ome/omeroweb-install/pull/11/commits/fd28e129b6dabfecad9a85c6431cb64fc401aa0f#diff-724be3e42d43d54ceecbaddf3f461850L10 ?

**Comment by jburel** [comment]
I have merged https://github.com/openmicroscopy/openmicroscopy/pull/5153
could you change travis back to point to latest?

**Comment by jburel** [comment]
Travis is green, merging so we can have an updated version of the walkthroughs

**Pull request merged**